protocol.md

```diff
@@ -317,8 +317,47 @@ The value of the  `Upload-Expires` header MUST be in
 
 ### Checksums
 
-This extension will define how to provide per file or per chunk checksums for
-uploaded files.
+Clients and servers MAY implement and use this extension to verify data
+integrity per chunk. In this case the server MUST add the `checksum` element to
+the `TUS-Extension` header.
+
+A client MAY include the `Content-MD5` header and its appropriate value in a
```

Comment:
I think I would just say that RFC 2616 defines how to compute and represent the checksum and drop a link to http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.15. Just as it's done already regarding trailers.